doc/source/dev/reviewer_guidelines.rst

```diff
@@ -116,4 +115,77 @@ the PR page.
 Assuming you have your :ref:`development environment<development-environment>`
 set up, you can now build the code and test it.
 
+.. _saved-replies:
+
+Standard replies for reviewing
+==============================
+
+It may be helpful to store some of these in GitHub's `saved
+replies <https://github.com/settings/replies/>`_ for reviewing:
+
+**Usage question**
+    ::
+
+        You are asking a usage question. The issue tracker is for bugs and new
+        features. I'm going to close this issue, feel free to ask for help via
+        our [help channels](https://numpy.org/gethelp/).
+
+**You’re welcome to update the docs**
+    ::
+
+        Please feel free to offer a pull request updating the documentation if
+        you feel it could be improved.
+
+**Self-contained example for bug**
+    ::
+
+        Please provide a [self-contained example code]
+        (https://stackoverflow.com/help/mcve), including imports and data
+        (if possible), so that other contributors can just run it and reproduce
+        your issue. Ideally your example code should be minimal.
+
+**Software versions**
+    ::
+
+        To help diagnose your issue, please paste the output of:
+        ```py
```

Comment:
Stray "py" ?

Reply:
This was meant to highlight the code as python in a markdown comment... mmmhhh,  this is a shell command line and after checking the rendering, I don't believe it increase readability... I have removed it.